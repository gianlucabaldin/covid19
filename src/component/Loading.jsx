import React from 'react';
import ClipLoader from 'react-spinners/ClipLoader';

const Loading = ({ loading = true }) => (
  <ClipLoader
    color="#303f9f"
    loading={loading}
    size={40}
    data-id="loading-spinner"
  />
);

export default Loading;
