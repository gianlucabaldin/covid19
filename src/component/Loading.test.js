import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

describe('Loading component', () => {
  const wrapper = shallow(<Loading />);
  it('should renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
