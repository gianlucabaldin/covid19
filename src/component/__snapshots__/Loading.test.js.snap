// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should renders properly 1`] = `
<Loader
  color="#303f9f"
  css=""
  data-id="loading-spinner"
  loading={true}
  size={40}
/>
`;
